mesh/mesh_readers.cpp

```diff
@@ -485,37 +487,38 @@ void Mesh::CreateVTKMesh(const Vector &points, const Array<int> &cell_data,
 
       // The following loop reorders pts_dofs so vertices are visited in
       // canonical order
-
-      // Keep the original ordering of the vertices
-      int i, n;
-      for (n = i = 0; i < np; i++)
       {
-         if (pts_dof[i] != -1)
+         // Keep the original ordering of the vertices
+         int n = 0;
```

Comment:
Rather than adding a new scope here, I think `n` can be replaced with `NumOfVertices`

Reply:
Done